Source/DataSources/GeoJsonDataSource.js

```diff
@@ -851,17 +845,7 @@ define([
         });
     };
 
-    function load(that, geoJson, options, sourceUri) {
```

Comment:
We still need to pass `sourceUri` in to the function to get the name of the file.